assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -242,6 +242,70 @@ describe("DiversionPage", () => {
     expect(screen.queryByTitle("Detour End")).toBeNull()
   })
 
+  test("shows 'Regular Route' text when the detour is finished", async () => {
+    jest.mocked(fetchDetourDirections).mockResolvedValue(
+      ok(
+        detourShapeFactory.build({
+          directions: [
+            { instruction: "Turn left on Main Street" },
+            { instruction: "Turn right on High Street" },
+            { instruction: "Turn sharp right on Broadway" },
+          ],
+        })
+      )
+    )
+
+    jest
+      .mocked(fetchFinishedDetour)
+      .mockResolvedValue(finishedDetourFactory.build())
+
+    const { container } = render(<DiversionPage />)
+
+    await act(async () => {
+      fireEvent.click(originalRouteShape.get(container))
+    })
+
+    await act(async () => {
+      fireEvent.click(originalRouteShape.get(container))
+    })
+
+    expect(await screen.findByText("Regular Route")).toBeVisible()
+  })
+
+  test("does not show 'Regular Route' when detour not finished", async () => {
```

Comment:
```suggestion
  test("does not show 'Regular Route' when detour is not finished", async () => {
```

Nit: grammar

Reply:
🤔 Twas not a typo, but if this makes it meaningfully more readable to you, might as well fix.

Reply:
WELP!

I don't feel strongly about it, but I _think_ most of the test titles read more like full sentences?

Reply:
I'm not sure we can/should aim for unanimity in sentence structure across devs for test titles, but I'll commit this one since all I have to do is click a button!